libs/application/templates/driving-instructor-registrations/src/fields/StudentsOverview/index.tsx

```diff
@@ -0,0 +1,182 @@
+  }
+
+  useEffect(() => {
+    filter(searchTerm)
+  }, [data?.drivingBookStudentListByTeacherSsn?.data, searchTerm])
+
+  return (
+    <Box marginBottom={10}>
+      {showTable ? (
+        <Stack space={5}>
+          <Box
+            display={['block', 'flex', 'flex']}
+            justifyContent={'spaceBetween'}
+          >
+            <Box width="half" className={styles.mobileWidth}>
+              <Input
+                name="searchbar"
+                placeholder={formatMessage(m.studentsOverviewSearchPlaceholder)}
+                icon="search"
+                backgroundColor="blue"
+                size="sm"
+                value={searchTerm}
+                onChange={(e) => setSearchTerm(e.target.value)}
+              />
+            </Box>
+            <FindStudentModal
+              application={application}
+              setShowTable={setShowTable}
+              setStudentId={setStudentId}
+            />
+          </Box>
+          <T.Table>
+            <T.Head>
+              <T.Row>
+                <T.HeadData>
+                  {formatMessage(m.studentsOverviewTableHeaderCol1)}
+                </T.HeadData>
+                <T.HeadData>
+                  {formatMessage(m.studentsOverviewTableHeaderCol2)}
+                </T.HeadData>
+                <T.HeadData box={{ textAlign: 'center' }}>
+                  {formatMessage(m.studentsOverviewTableHeaderCol3)}
+                </T.HeadData>
+                <T.HeadData></T.HeadData>
+              </T.Row>
+            </T.Head>
+            <T.Body>
+              {pageStudents && pageStudents.length ? (
+                pageStudents.map((student, key) => {
+                  return (
+                    <T.Row key={key}>
+                      <T.Data>{student.name}</T.Data>
+                      <T.Data>{student.ssn}</T.Data>
+                      <T.Data box={{ textAlign: 'center' }}>
+                        {student.totalLessonCount}
+                      </T.Data>
+                      <T.Data>
+                        <Button
+                          variant="text"
+                          size="small"
+                          onClick={() => {
+                            setStudentId(student.ssn)
+                            setShowTable(false)
+                          }}
+                        >
+                          {formatMessage(m.studentsOverviewRegisterHoursButton)}
+                        </Button>
+                      </T.Data>
+                    </T.Row>
+                  )
+                })
+              ) : (
+                <T.Row>
+                  <T.Data>
+                    {formatMessage(m.studentsOverviewNoStudentFound)}
+                  </T.Data>
+                  <T.Data></T.Data>
+                  <T.Data></T.Data>
+                  <T.Data></T.Data>
+                </T.Row>
+              )}
+            </T.Body>
+          </T.Table>
+          {pageStudents && !!pageStudents.length && (
```

Comment:
Is there a reason we are using `!!pageStudents.length` instead of `pagesStudents.length > 0?`